docs/reference/field-ref.asciidoc

```diff
@@ -127,30 +130,27 @@ These fields must be mapped to display HTTP request data in the {es-sec-app}:
 [discrete]
 ==== TLS fields
 
-This field must be mapped to display TLS data in the {es-sec-app}:
+{es-sec} relies on this field to analyze and display TLS data:
 
 * `tls.server.hash.sha1`
 
-These fields can be mapped to display additional TLS data in the {es-sec-app}:
+{es-sec} may also use these fields to analyze and display TLS data:
 
 * `tls.server.issuer`
 * `tls.server.ja3s`
 * `tls.server.not_after`
 * `tls.server.subject`
 
 [float]
-== Event and external alert fields
+== Fields required for events and external alerts
+{es-sec} relies on these fields and values to analyze and display event and external alert data:
 
-These fields must be mapped to display event and external alert data in the
-{es-sec-app}:
-
-* `@timestamp`
 * `event.kind`
+
 +
-NOTE: For external alerts, the `event.kind` field value must be `alert`.
+NOTE: For external alerts, the `event.kind` field's value must be `alert`.
```

Comment:
Does Security just _rely_ on the `event.kind` field with a value of `alert` and the subsequent fields in the list Security _may_ also use those to analyze and display event and external alert data? Just want to make sure 1) we are accurate and 2) we are not repeating ourselves unnecessarily.